imagetest/testworkflow.go

```diff
@@ -140,6 +141,7 @@ func (t *TestWorkflow) appendCreateVMStep(name, hostname string) (*daisy.Step, e
 	instance.Metadata = make(map[string]string)
 	instance.Metadata["_test_vmname"] = name
 	instance.Metadata["_test_package_url"] = "${SOURCESPATH}/testpackage"
+	instance.Metadata["_ssh_key_url"] = "${SOURCESPATH}/ssh-key"
```

Comment:
this does suggest only one ssh key can exist in a single test workflow. might consider naming the key after the user in some way?

Reply:
On way I can think to make per user ssh-key is to move this into setup where we generate keys. During setup we could decide how many keys needs.